### PR TITLE
SH-MG-3: Three more miracles of George

### PR DESCRIPTION
**Pull request by SusanneHummel**
Here come three more miracles, all in all now 22 out of 42

slowly but surely ...

or as we say in German "mühsam ernährt sich das Eichhörnchen"

### CODE REVIEW

File: new/LIT6234MGBebaPalm.xml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6234MGBebaPalm" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of St George: A malicious appointee from
+                    Bǝbā, who hates St George, is killed and hung on a palm tree</title>
+                <title xml:id="t2" xml:lang="en">How a man hated the saint was killed</title>
```
Comment (Gnisci) [inline new/LIT6234MGBebaPalm.xml:10]: man "who" hated the saint

File: new/LIT6236MGEucharist.xml [hunk 2]
```diff
@@ -0,0 +1,102 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6236MGEucharist" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Miracle of St George: A pagan woman takes the
+                    Eucharist and dies a suffering death</title>
```
Comment (Gnisci) [inline new/LIT6236MGEucharist.xml:9]: an agonizing/painful death

**Review by Gnisci — CHANGES_REQUESTED** [review]

File: new/LIT6235MGOldCairo.xml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            <div type="bibliography">
+
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusText"/>
+                        <citedRange unit="page">15–18 (21th miracle)</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusTrans"/>
+                        <citedRange unit="page">9–10 (21th miracle)</citedRange>
```
Comment (MassimoVilla) [inline new/LIT6235MGOldCairo.xml:67]: 21st miracle

File: new/LIT6235MGOldCairo.xml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusText"/>
+                        <citedRange unit="page">15–18 (21th miracle)</citedRange>
```
Comment (MassimoVilla) [inline new/LIT6235MGOldCairo.xml:63]: 21st miracle

File: new/LIT6235MGOldCairo.xml [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+                        <ptr target="bm:Arras1953GeorgiusTrans"/>
+                        <citedRange unit="page">9–10 (21th miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:Strelcyn1974CatJRylands"/>
+                        <citedRange unit="page">35</citedRange>
+                        <citedRange unit="item">Ms 12, II.22 (21th miracle)</citedRange>
```
Comment (MassimoVilla) [inline new/LIT6235MGOldCairo.xml:75]: 21st

File: new/LIT6235MGOldCairo.xml [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:GrebTiss1935Codices"/>
+                        <citedRange unit="page">721</citedRange>
+                        <citedRange unit="item">Vat 233, 3.22 (21th miracle)</citedRange>
```
Comment (MassimoVilla) [inline new/LIT6235MGOldCairo.xml:80]: 21st

File: new/LIT6236MGEucharist.xml [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusText"/>
+                        <citedRange unit="page">18–20 (22th miracle)</citedRange>
```
Comment (MassimoVilla) [inline new/LIT6236MGEucharist.xml:63]: 22nd

File: new/LIT6236MGEucharist.xml [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusTrans"/>
+                        <citedRange unit="page">10–11 (22th miracle)</citedRange>
```
Comment (MassimoVilla) [inline new/LIT6236MGEucharist.xml:67]: 22nd

File: new/LIT6236MGEucharist.xml [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+                    </bibl>
+                </listBibl>
+
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:Strelcyn1974CatJRylands"/>
+                        <citedRange unit="page">35</citedRange>
+                        <citedRange unit="item">Ms 12, II.23 (22th miracle)</citedRange>
```
Comment (MassimoVilla) [inline new/LIT6236MGEucharist.xml:75]: 22nd

File: new/LIT6236MGEucharist.xml [hunk 10]
```diff
@@ -0,0 +1,102 @@
...
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:GrebTiss1935Codices"/>
+                        <citedRange unit="page">721</citedRange>
+                        <citedRange unit="item">Vat 233, 3.23 (22th miracle)</citedRange>
```
Comment (MassimoVilla) [inline new/LIT6236MGEucharist.xml:80]: 22nd

File: new/LIT6236MGEucharist.xml [hunk 11]
```diff
@@ -0,0 +1,102 @@
...
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BDLaethb4"/>.</note>
+                    <ab>፳፪፡ ተአምሪሁ፡ <gap reason="ellipsis" resp="SH"/> ወሀለወት፡ በሀገር፡ እንተ፡ ዘከርናሃ፡ ቀዲሙ፡
+                        ብእሲት፡ አረማዊት፡ ወሰይጣን፡ ቦአ፡ ውስቴታ። ወይእቲ፡ መጽአት፡ ኀበ፡ ቤተ፡ ክርስቲያን፡</ab>
+                    <div type="textpart" subtype="explicit" xml:lang="gez">
+                        <note>This explicit is taken from <ref type="mss" corresp="BDLaethb4"
+                            />.</note>
+                        <ab>ወዘከመ፡ ሞተት፡ በዐቢይ፡ ምንዳቤ። ወእሙንቱሰ፡ ሶበ፡ ሰምዑ፡ ዘንተ፡ ኵሎ፡ አንከሩ፡ ጥቀ። ወሰብሕዎ፡
+                            ለእግዚአብሔር፡ ስቡሕ፡ ወልዑል። ወለቅዱስ፡ ማር፡ ጊዮርጊrb ስ፡ ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስለ፡ <gap
```
Comment (MassimoVilla) [inline new/LIT6236MGEucharist.xml:94]: ጊዮርጊስ፡ ?

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Sanne!

File: new/LIT6235MGOldCairo.xml [hunk 12]
```diff
@@ -0,0 +1,102 @@
...
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BDLaethb4"/>.</note>
+                    <ab>፳፩፡ ተአምሪሁ፡ <gap reason="ellipsis" resp="SH"/> ወካዕበ፡ ካልዓ፡ እዜንወክሙ፡ ተአምረ፡ ዘገብረ፡
+                        መንክረ፡ ቅዱስ፡ ሰማዕት፡ ማር፡ ጊዮርጊስ፡ በውስተ፡ ቤተ፡ ክርስቲያኑ፡ ካልዕት፡ እንተ፡ ተሐንጸት፡ ሎቱ፡</ab>
+                    <div type="textpart" subtype="explicit" xml:lang="gez">
+                        <note>This explicit is taken from <ref type="mss" corresp="BDLaethb4"
+                            />.</note>
+                        <ab>ወዜነዎሙ፡ ለኵሎሙ፡ ደቂቀ፡ መርጡል፡ ኵሎ፡ ዘነገሮ፡ ውእቱ፡ አረሚ፡ ወሰብሖ፡ ለእግዚአብሔር፡ ስቡሕ፡
+                            ወልዑል፡ ገባሬ፡ ተአምራት፡ በደ፡ ሰማዕት፡ ቅዱስ፡ ጊዮርጊስ፡ ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስለ፡ <gap
```
Comment (thea-m) [inline new/LIT6235MGOldCairo.xml:94]: በደ in the ms?

File: new/LIT6236MGEucharist.xml [hunk 13]
```diff
@@ -0,0 +1,102 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6236MGEucharist" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Miracle of St George: A pagan woman takes the
```
Comment (thea-m) [inline new/LIT6236MGEucharist.xml:8]: en

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

**Comment by thea-m** [comment]
More than half of them done now!!

File: new/LIT6235MGOldCairo.xml (hunk 12 above)
Comment (SusanneHummel) [reply new/LIT6235MGOldCairo.xml]: በእደ፡

**Review by SusanneHummel — COMMENTED** [review]
Thank to all! I incorpoartted all your ssuggestions.

**Review by MassimoVilla — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**